tests/FSharp.Compiler.ComponentTests/FSharp.Compiler.ComponentTests.fsproj

```diff
@@ -263,6 +263,7 @@
   <ItemGroup>
     <Content Include="resources\**" CopyToOutputDirectory="Never" CopyToPublishDirectory="PreserveNewest" />
     <EmbeddedResource Remove="Properties\**" />
+    <None Remove="EmittedIL\GenericComparison\Hash09.fsx.il.bsl" />
```

Comment:
Do we need .bsl within .fsproj?

Reply:
The baselines are displayed in the ui, even though they aren't added persisted.  VS adds this remove, when you do a delete.  I will delete the remove this line..